server/game/cards/characters/07/maryaseaworth.js

```diff
@@ -4,16 +4,20 @@ class MaryaSeaworth extends DrawCard {
     setupCardAbilities(ability) {
         this.reaction({
             when: {
-                onBypassedByStealth: (event, challenge, source, target) => {
-                    this.bypassed = target;
-                    return true;
+                onChallengeInitiated: event => { 
```

Comment:
This implementation is no longer correct - previously, if 2 characters were bypassed by stealth, 2 events would be fired and Marya would trigger twice. However, now she will only trigger once since onChallengeInitiated is only fired once. This... makes things very difficult, given that she would need to pay 1 gold for each character, and the limit is now associated with challenges being initiated, not characters bypassed.